streams/src/main/java/org/apache/kafka/streams/state/WindowStore.java

```diff
@@ -27,8 +27,11 @@
 public interface WindowStore<K, V> extends StateStore, ReadOnlyWindowStore<K, V> {
 
     /**
-     * Put a key-value pair with the current record time as the timestamp
-     * into the corresponding window
+     * Use the current record timestamp as the {@code windowStartTimestamp} and
+     * delegate to {@link WindowStore#put(Object, Object, long)}.
+     *
+     * It's highly recommended to use {@link WindowStore#put(Object, Object, long)} instead.
```

Comment:
nit: maybe add small comment why `WindowStore#put(Object, Object, long)` is preferred?